lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,12 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, options = {})
+          domain = default_sbg_domain_options.merge(options.assert_valid_keys(:begin, :end, :outcome, :key, :primary))
```

Comment:
Roqua gebruikt from en till in answers en invitations. start en stop in prot_subs.
Ik zou zeggen laten we from en till gebruiken voor consistency, mogelijk met een prefix. 

Ook zou ik graag key defaulten naar sbg_key of verplichten, zodat we elders geen defaults hoeven toe te voegen.